SeleniumSolarTechnology/SpecFlowSeleniumSolarTechnology/Features/Cart.feature

```diff
@@ -0,0 +1,14 @@
+﻿Feature: Cart
+
+@mytag
+Scenario: Add and remove products from cart
+	    Given Open Solartechnology Shop page
+    And Open Cable And Switching link
+	    When Add product item to cart(1)
+	And Make order
+    And Remove product item from cart(0)
```

Comment:
not clear what this number means. Why  Add product item to cart(1) but Remove product item from cart(0)
Are these some different indexes? Or we add one product and remove another...
Test case should be clear, so anyone can understand what it does